WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -176,6 +176,9 @@ public WpkhDescriptors GetOutputDescriptors(string password, Network network)
 	[JsonProperty(Order = 15, PropertyName = "IsCoinjoinProfileSelected")]
 	public bool IsCoinjoinProfileSelected { get; set; } = false;
 
+	[JsonProperty(Order = 16, PropertyName = "CoinjoinProfile")]
+	public string CoinjoinProfile { get; set; }
```

Comment:
Instead of storing the name, couldn't decide which one is selected by the values of the current profile? I would find that more cleaner

Reply:
@soosr I tried that at first, but I can't do an exact match because `PrivateCoinJoinProfileViewModel` uses a random value.

We also can't safely assume that Maximize Privacy is selected if `anonScore > 50` , because there's no way to determine if user has manually modified that value.

Is there any other way to infer this?